etc/config.xml

```diff
@@ -86,6 +86,8 @@ tr.shipping.totals td.amount span.price,
                 <catalog_ingestion_instant_enabled>1</catalog_ingestion_instant_enabled>
                 <catalog_ingestion_instant_async_enabled>0</catalog_ingestion_instant_async_enabled>
                 <catalog_ingestion_instant_events>product_stock_status</catalog_ingestion_instant_events>
+                <instant_button_variant></instant_button_variant>
+                <instant_button_variant_ppc></instant_button_variant_ppc>
```

Comment:
What do you think about PPC as default here?

Reply:
It doesnt exist by default. 
By default merhcants use either blank or default because only those exist. Then they can get a variant created with a label and put this label as query parameter &variant={{variant name}}
So theres no guarentee it will be called PPC

Reply:
Let's discuss this with product team.